app/parsers/bulkrax/application_parser_decorator.rb

```diff
@@ -0,0 +1,16 @@
+# OVERRIDE Bulkrax 4.4.1 to get a patch for create_objects
+
+module Bulkrax
+  module ApplicationParser
+    module ApplicationParserDecorator
+      # override create_objects to get the patch from https://github.com/samvera-labs/bulkrax/pull/663
+      def create_objects(types = [])
+        types.each do |object_type|
```

Comment:
1. Without the `DEFAULT_OBJECT_TYPES` variable added by https://github.com/samvera-labs/bulkrax/pull/663/files, what does `types` end up being here? 
2. Also, can we please add a `TODO` comment that says to delete this decorator when upgrading Bulkrax to v5.x.x?

Reply:
I realized i just got the PR in the comment wrong. PR #663 IS already in version 4.4.1, its this patch PR https://github.com/samvera-labs/bulkrax/pull/713 that we needed to pull in.

Reply:
updating comments now though to also add the todo and fix the PR that is linked there!

Reply:
@bkiahstroud done!!